src/poprox_storage/repositories/newsletters.py

```diff
@@ -56,6 +57,41 @@ def store_newsletter(self, newsletter: Newsletter):
                 )
                 self.conn.execute(stmt)
 
+    def store_newsletter_feedback(self, account_id: list[UUID], newsletter_id: list[UUID], ispositive: bool):
+        newsletter_table = self.tables["newsletters"]
+
+        self.conn.commit()
```

Comment:
@karlhigley I _think_ I remember hearing that you had moved to prefer commits not occur inside repositories, am I remembering that right?

Reply:
If you need transaction management, then yes. Here it's just one update that will be an atomic change anyway, so I don't think we need to `self.conn.begin()` in this method, which means we can also avoid the pre-emptive `commit()` that gets us outside sqlalchemy's automatic transaction management